tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+envlist = py27, py35, py36, flake8
+
+[travis]
+python =
+    3.6: py36
+    3.5: py35
+    2.7: py27
+
+[testenv:flake8]
+basepython = python
+deps = flake8
+commands = flake8 barril
+
+[testenv]
+setenv =
+    PYTHONPATH = {toxinidir}
+deps =
+    -r{toxinidir}/requirements_dev.txt
```

Comment:
We should not use `requirements` files for libraries. Please declare all dependencies and extras on `setup.py`.